dali/python/nvidia/dali/_multiproc/pool.py

```diff
@@ -298,15 +298,15 @@ def __init__(self, mp, workers_contexts : List[WorkerContext], result_queue : Sh
         start_method = mp.get_start_method()
         if not workers_contexts:
             raise RuntimeError("Cannot start a pool with no workers")
-        if start_method == 'fork' and _b.HasCudaContext():
+        if start_method == 'fork' and _b.IsDriverInitialized():
             raise RuntimeError(
                 "Error when starting Python worker threads for DALI parallel External Source. "
-                "Cannot fork a process when there is a CUDA context already bound to the process. "
-                "CUDA context is acquired during ``Pipeline.build()``, or can be acquired by another "
+                "Cannot fork a process when the CUDA has been initialized in the process. "
+                "CUDA is initialized during ``Pipeline.build()``, or can by another "
```

Comment:
Isn't some verb needed after can?
or _may be initialized_ by another

Reply:
Done